protos/events.proto

```diff
@@ -27,16 +27,14 @@ package protos;
 enum EventType {
         REGISTER = 0;
         BLOCK = 1;
-	GENERIC = 2;
-	CHAINCODE = 3;
+	CHAINCODE = 2;
```

Comment:
Protobuf renumbers can impact the ABI and should not be taken lightly.  Do we know if there were any clients that might have been consuming GENERIC?  My assumption is "no" but good to check.